plugins/ufm_log_analyzer_plugin/src/loganalyze/log_analyzer.py

```diff
@@ -343,33 +343,48 @@ def create_analyzer(parsed_args, full_extracted_logs_list,
         # Next section is to create a summary PDF
         pdf_path = os.path.join(args.destination, "UFM_Dump_analysis.pdf")
         pdf_header = (
-            f"Dump analysis for {os.path.basename(args.location)}, hours={args.hours}"
+            f"{os.path.basename(args.location)}, hours={args.hours}"
         )
-        FABRIC_INFO = str(ibdiagnet_analyzer.get_fabric_size() \
-                        if ibdiagnet_analyzer else "No Fabric Info found")
+        fabric_info = ibdiagnet_analyzer.get_fabric_size() \
+                        if ibdiagnet_analyzer else "No Fabric Info found"
 
-        LINK_FLAPPING = str(links_flapping_analyzer.get_link_flapping_last_week() \
-                            if links_flapping_analyzer else "No link flapping info")
+        link_flapping = links_flapping_analyzer.get_link_flapping_last_week() \
+                            if links_flapping_analyzer else "No link flapping info"
         # PDF creator gets all the images and to add to the report
-        TEXT = FABRIC_INFO + os.linesep + "Link Flapping:" + os.linesep + LINK_FLAPPING
-        pdf = PDFCreator(pdf_path, pdf_header, png_images, TEXT)
+        text = str(fabric_info) + os.linesep + "Link Flapping:" + os.linesep + str(link_flapping) # pylint: disable=invalid-name
+
+        critical_events_burst = event_log_analyzer.get_critical_event_bursts()
+        critical_events_text = "The minute           event_type     event    count" # pylint: disable=invalid-name
+        for critical_event in critical_events_burst:
+            timestamp = critical_event['timestamp']
+            event_type = critical_event['event_type']
+            event = critical_event['event']
+            counter = critical_event['count']
+            event_text = f"{timestamp} {event_type} {event} {counter}"
+            critical_events_text = critical_events_text + os.linesep + event_text

```

Comment:
this header is going to be above each event_text?

Reply:
What do you mean when you say header?

It prints the "title" and then each line with the data